File: 1189. Maximum Number of Balloons.py
class Solution:
    def maxNumberOfBalloons(self, s: str) -> int:
        d = collections.Counter(s)
        return min(min([d.get(c, 0) for c in "ban"]), min([d.get(c, 0)//2 for c in "lo"]))
        
        

File: 1189. Maximum Number of Balloons.cpp
class Solution {
public:
    int maxNumberOfBalloons(string s) {
        unordered_map<char, int> m;
        for (auto c : s) ++m[c];
        int ans = INT_MAX;
        string once="ban", tw="lo";
        for (auto c : once) ans = min(ans, m[c]); 
        for (auto c : tw) ans = min(ans, m[c]/2);
        return ans;
    }
};